README.md

```diff
@@ -132,6 +132,30 @@ const validator = (value) => {
     // Parse the value, modifying it
     return value.replace('aa', 'bb');
 }
+
+const asyncValidator = async (value) => {
```

Comment:
The example seems a bit too complicated, can we make it simple by just awaiting “someAsyncAssert”, or similar?

Reply:
I m up for both approach, I tried to add a remote fetch to make it more 'real' but if you prefer a simpler like:

```
async (value) => {
  await myfunc();
  return value;
}
```


You prefer the last one?

Reply:
yep!

Reply:
Here you are Sir!